package.json

```diff
@@ -9,7 +9,7 @@
   },
   "author": "Expo",
   "engines": {
-    "node": ">=12"
+    "node": ">=22"
```

Comment:
I think this is too big of a jump potentially, since we use this library in CLIs, which adhere to the Node support cycle: https://nodejs.org/en/about/previous-releases. Later this month, Node 18 will drop off and we'll support just Node 20+.

Unstable fetch was added to Node 18 I believe and it became stable in 21. However, I'm not sure we rely on the differences between Node 20's fetch and Node 21's in this library; if we could set this to `node >= 20` that'd be a good compromise.